modules/ROOT/pages/restarting-a-node.adoc

```diff
@@ -1,78 +1,118 @@
+```
 +
-----
-sudo systemctl stop firewalld
-sudo systemctl disable firewalld
-sudo systemctl mask firewalld
-----
-. Retrieve the *name* of target node:
+If the output from the previous command lists any pod not in "Running" state, then fix all issues before proceeding.
+
+=== Install Updates
+
+After verifying that the cluster is healthy, select a node on which to perform the following steps:
+
+. Retrieve the *name* of the target node:
 +
-----
+```
 kubectl get nodes
-----
+```
 . Drain the node:
 +
-----
+```
 kubectl drain --delete-local-data --ignore-daemonsets <node_name>
-----
-. Stop Gravity
+```
+. After the node is drained, identify the gravity unit name:
 +
-----
-systemctl list-units | grep "planet-master"
-  # note the full name of the Gravity "planet-master" unit
-systemctl stop <garvity_unit_name>
-----
+```
+sudo systemctl list-units --plain | awk '/planet-master/{print $1}'
+```
+The previous command outputs a `systemd` unit name, which is used as the `$unit_name` value in the next step.
+. Stop the gravity unit:
++
+```
+systemctl stop $unit_name
+systemctl status $unit_name
+```
+. After the `status` subcommand indicates the gravity unit is stopped, install the needed updates and reboot the server.
 
-== Apply Node Maintenance
+=== Perform Post-update Check
 
-Once the node has been drained and cordoned, any desired maintenance or a system reboot may be performed.  Caution must be used, however, to avoid any updates to the system that violate the requirements of Anypoint Platform PCE.
+. Verify that all systems correctly restarted:
++
+```
+systemctl status --failed
+```
+The previous command should return zero units, which means that your system boot phase was successful and all units started. If any units failed to start, fix all issues before proceeding.
+. Check the gravity `systemd` units:
++
+```
+systemctl list-units '*gravity__gravitational*'
+```
+. Check for errors in the logs by running `journalctl`.
 
-== Rejoin the Node
+. If everything started correctly, run `gravity enter` and verify the results.
 
-. Check Gravity Planet Master and Teleport status:
+. Check the cluster status again:
 +
-----
-systemctl list-units | grep gravity__gravitational
-----
-. If either Gravity service is not “loaded active running”, then enable it:
+```
+gravity status
+etcdctl cluster-health
+gravity get pod -o wide--all-namespaces | grep -vi Running
+```
+
+=== Rejoin the Node
+
+. Check the gravity Planet master and teleport status:
 +
-----
-systemctl enable <name_of_the_gravity_unit>
-----
+```
+systemctl list-units '*gravity__gravitational*'
+```
+
+. If the status of the gravity service is not `loaded`, `active`, or `running`, then you must enable it:
++
+```
+systemctl enable --now <name_of_the_gravity_unit>
+```
+
 . Uncordon the node:
 +
-----
+```
 kubectl uncordon <node_name>
-----
+```
+The target node is set to again schedule pods. This can take a few minutes to complete.
```

Comment:
Suggested moving this definition of "uncordon" to the step action itself. That leaves...:
```suggestion
This can take a few minutes to complete.
```